.meta/_partials/fields/_tls_common.toml

```diff
@@ -0,0 +1,80 @@
+[<%= namespace %>.tls.children.ca_path]
+type = "string"
+examples = ["/path/to/certificate_authority.crt"]
+groups = <%= groups.to_toml %>
+<%= relevant %>
+description = """\
+Absolute path to an additional CA certificate file, in DER or PEM format \
+(X.509). \
+Only one of this and `ca_text` may be set.
+"""
+
+[<%= namespace %>.tls.children.ca_text]
```

Comment:
Seems like it'd be possible for us to parse the `ca` field and if it's a Path then read from it, and if it's text read it as a CA, no?

Reply:
What you describe is possible. However, before this change there was no generic `ca` field, only `ca_path`, which directed the solution to having two different fields (putting inline text into a "path" doesn't make much sense).